build-system/tasks/prepend-global/index.js

```diff
@@ -141,8 +142,8 @@ async function applyConfig(
 ) {
   await checkoutBranchConfigs(filename, opt_localBranch, opt_branch);
 
-  let [configString, targetString, overlayString] = await Promise.all([
-    fs.promises.readFile(filename, 'utf8'),
+  let configString = fs.promises.readFile(filename, 'utf8');
```

Comment:
This looks incorrect. One of the three calls to `fs.promises` is no longer being `await`ed. Also, failure to read `filename` will now throw an error (it didn't earlier).

Reply:
You're right about the missing await; that was a mistake I introduced when resolving a linting error around let/const. As for errors, failure to read `filename` would always throw an error. The only file read that is caught is for `customConfigFile`, no other file read failure was handled previously.